### ISSUE TITLE
Improve accessibility of Member blocks

### ISSUE THREAD

**Issue by danieltott** [Virtual-Coffee/virtualcoffee.io#436]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Context

The member profiles on https://virtualcoffee.io/members could use some a11y TLC:

- `alt` for the profile images isn't great. We probably can simply set this to `alt=""`
- Names are `h5` - out of heading order
- Redundant links - we're linking the member name as well as dropping a text link under the body - probably don't need both
- The `website` svg is missing the `img` role and a `<title>` element

### Proposed solution

- [ ] Set `alt=""` for the profile image
- [ ] Remove link inside member name
- [ ] Set member name to be `h3` and style back to match (ie it visually should look like it does now, but with an `h3` element instead of an `h5`
- [ ] Add `role="img"` to `website.svg`
- [ ] Add `<title>Website</title>` to `website.svg`

### Code of Conduct

- [X] I've read the Code of Conduct and understand my responsibilities as a member of the Virtual Coffee community

**Comment by colabottles** [comment]
I'll take this one.
